packages/react-components/review/beta/react-components.api.md

```diff
@@ -813,6 +813,54 @@ export const DEFAULT_COMPONENT_ICONS: {
     ContextMenuSpeakerIcon: JSX.Element;
 };
 
+// Warning: (ae-internal-missing-underscore) The name "DeviceMenuProps" should be prefixed with an underscore because the declaration is marked as @internal
```

Comment:
Fix underscore warnings (all `@internal` interfaces should be prefixed with `_`)